benefits/admin.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+from django.conf import settings
+from loguru import logger
+
+
+def pre_login_user(user, request):
```

Comment:
This is now a duplicate of what is in `benefits/core/admin.py`? The entire file should probably be deleted right?

Reply:
I agree with this. I hadn't realized the file wasn't deleted